pkg/api/v1alpha1/gitopsconfig_types.go

```diff
@@ -35,6 +37,10 @@ type Github struct {
 	Personal bool `json:"personal,omitempty"`
 }
 
+func (g *Github) ClusterRootPath() string {
```

Comment:
Why did we end up introducing this?

Reply:
to support the new gitops file structure. https://github.com/aws/eks-anywhere/issues/355 we move `flux-system` dir one level up